app/domain_models/common_error.rb

```diff
@@ -0,0 +1,85 @@
+# User Login
```

Comment:
I see it in Wayne's branch.

Reply:
Yes, we did the file seperation together. The reason we put it in both of our branches is we want both our branch to function normally.